backend/routes/event_payment_routes/admin_refund_routes.py

```
@@ -0,0 +1,873 @@
+    request: Request,
+    process_data: Dict[str, Any] = Body(...)
+):
+    """Process a refund request (approve/reject) - Admin only"""
+    try:
+        admin_uid = getattr(request.state, 'uid', None)
+        if not admin_uid:
+            raise HTTPException(status_code=401, detail="Authentication required")
+        
+        refund_request = await get_refund_request_by_id(request_id)
+        if not refund_request:
+            raise HTTPException(status_code=404, detail="Refund request not found")
```

Ruff (0.14.2)

Abstract `raise` to an inner function

(TRY301)